src/header/js/header-menus.js

```diff
@@ -89,112 +89,583 @@ export default ({variant = 'desktop'}) => {
+															data: {filled: `${item.tiles.slice(0, 4).length}`}
 														},
-														childitem.label
-													);
-
-													if (childitem.data) {
-														$($sublink, {
-															data: childitem.data
-														});
+														/* Global Navigation: Menus: Sublink
+														/* ============================== */
+			
+														...item.tiles.slice(0, 4).map(
+															(childitem) => {
+																const $sublink = $('a',
+																	{
+																		class: `${prefix}-sublink--featured`,
+																		href: childitem.href
+																	},
+																	$renderSvgOrImg({imgDef: childitem.icon, imgClass: `${prefix}-sublink-image`, imgWidth: childitem.width, imgHeight: childitem.height}),
+																	$('span', {class: `${prefix}-sublink-text`},
+																		childitem.label
+																	)
+																);
+			
+																if (childitem.data) {
+																	$($sublink, {
+																		data: childitem.data
+																	});
+																}
+			
+																if (childitem.newContext) {
+																	$($sublink, {
+																		target: '_blank',
+																		rel: 'noopener'
+																	});
+																}
+			
+																return $('li', {class: `${prefix}-subitem--featured`},
+																	$sublink
+																);
+															}
+														)
+													)
+												
+											);
+										});
+									}
+								} else {
+									if (hasMenuItems) {
+										let col1 = undefined;
+										let col2 = undefined;
+										let col3 = undefined;
+										let col4 = undefined;
+
+										if (item.menus.length <= 9) {
+											col1 = $('div', {class: `${prefix}-sublist--col`},									
+												...item.menus.slice(0,9).map(
+													(childitem) => {
+														const $heading = (childitem.heading) ? $('p', {class: `${prefix}-heading--label`}, childitem.heading) : '';
+														const $sublink = $('a',
+															{
+																class: `${prefix}-sublink`,
+																href: childitem.href
+															},
+															childitem.label
+														);
+
+														if (childitem.data) {
+															$($sublink, {
+																data: childitem.data
+															});
+														}
+
+														if (childitem.newContext) {
+															$($sublink, {
+																target: '_blank',
+																rel: 'noopener'
+															});
+														}
+
+														if ($heading) {
+															return $('li', {class: `${prefix}-subitem ${prefix}-subitem--heading`}, $heading, $($sublink));
+														} else {
+															return $('li', {class: `${prefix}-subitem`}, $heading, $($sublink));
+														}
 													}
-
-													if (childitem.newContext) {
-														$($sublink, {
-															target: '_blank',
-															rel: 'noopener'
-														});
+												)
```

Comment:
This part needs to be refactored. It's way too repetitive. It could easily be changed to the following.
```js
col1 =  $('div', {class: `${prefix}-sublist--col`} //...
if (item.menus.length > 9) {
  col2 =  $('div', {class: `${prefix}-sublist--col`} //...
if (item.menus.length > 18) {
  col3 =  $('div', {class: `${prefix}-sublist--col`} //...
// etc.
```

This is better but you should create a method called something like `createColumn` instead. `createColumns` can take an array of columns and return a mapped version of them.
```js
col1 =  createColumn(...item.menus.slice(0,9))
if (item.menus.length > 9) {
  col2 = createColumn(...item.menus.slice(9,18))
if (item.menus.length > 18) {
 col3 =  createColumn(...item.menus.slice(18, 27))
// etc.
```

You can easily loop through `item.menus` too. As well as have a method called `createColumn`. This would be much better.

For Example:
Use a pointer variable starting at 0. Loop through `items.menus` and slice out the first 9. Then, do `pointer += 9`. 
```js
...items.menus.slice(pointer, pointer + 9)
```

Reply:
This is a really good idea, I agree